config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-apps.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# generates wg-k8s-infra app deployment job configs
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+readonly OUTPUT="${SCRIPT_DIR}/wg-k8s-infra-apps.yaml"
+# list of subdirs in kubernetes/k8s.io/apps
+readonly APPS=(
+    gcsweb
+    k8s-io
+    kubernetes-external-secrets
+    node-perf-dash
+    perfdash
+    prow
+    publishing-bot
+    sippy
+    slack-infra
+    triageparty-release
+)
+
+cat >"${OUTPUT}" <<EOF
+# DO NOT EDIT. Automatically generated by $0
+
+postsubmits:
+  kubernetes/k8s.io:
+EOF
+
+for app in "${APPS[@]}"; do
+    cat >>"${OUTPUT}" <<EOF
+    - name: post-k8sio-deploy-app-${app}
+      cluster: k8s-infra-prow-build-trusted
+      decorate: true
+      max_concurrency: 1
+      # intended for ignoring changes to README.md or OWNERS
+      run_if_changed: '^apps\/${app}\/(.*.yaml|deploy.sh)$'
+      branches:
+        - ^main$
+      annotations:
+        testgrid-create-test-group: 'true'
+        testgrid-dashboards: wg-k8s-infra-apps
+        testgrid-description: 'runs https://git.k8s.io/k8s.io/apps/${app}/deploy.sh if files change in kubernetes/k8s.io/apps/${app}'
+        testgrid-alert-email: k8s-infra-rbac-${app}k8s-infra-alerts@kubernetes.io, k8s-infra-alerts@kubernetes.io
+        testgrid-num-failures-to-alert: '1'
+      rerun_auth_config:
+        github_team_slugs:
+        # proxy for wg-k8s-infra-oncall
+        - org: kubernetes
+          slug: wg-k8s-infra-leads
+        # proxy for test-infra-oncall
+        - org: kubernetes
+          slug: test-infra-admins
+        # TODO: sig-specific team in charge of this app
+        # - org: kubernetes
+        #   slug: sig-foo-bar
```

Comment:
For followup: either we make a github team that can be easily guessed at by this script, or I'm going to need to make something uglier than the nice bash array this script uses